docs/operator-guides/security.md

```diff
@@ -38,9 +38,10 @@ You can secure access to Airbyte using the following methods:
 
 - Deploy Airbyte in a private network or use a firewall to filter which IP is allowed to access your host.
 - Deploy Airbyte behind a reverse proxy and handle the access control on the reverse proxy side.
+- Update the default username and password in your environment's `.env` file.
```

Comment:
```suggestion
- Update the default username and password in your environment's `.env` file:
    ```
		# Proxy Configuration
		# Set to empty values, e.g. "" to disable basic auth
		BASIC_AUTH_USERNAME=your_new_username_here
		BASIC_AUTH_PASSWORD=your_new_password_here
    ```
```